client/src/app/+admin/moderation/video-abuse-list/video-abuse-list.component.ts

```diff
@@ -119,6 +218,11 @@ export class VideoAbuseListComponent extends RestTable implements OnInit {
     return Account.CREATE_BY_STRING(account.name, account.host)
   }
 
+  setTableFilter (filter: string) {
+    const filterInput = document.getElementById('table-filter') as HTMLInputElement
```

Comment:
Why don't you use `ViewChild` to get the filter input?

Reply:
Because the element is usually not generated (it is within p-table's rendering) before the AfterContentChecked stage.